swingset/src/test/java/com/nqadmin/swingset/models/AbstractComboBoxListSwingModelTest.java

```diff
@@ -261,7 +261,7 @@ public void testCreateListItem() {
 		assertFalse(result.equals(result2));
 	}
 
-	private boolean getElemEquals(Object[] elems, int listItemIndex, LI.Remodel remodel) {
+	private static boolean getElemEquals(Object[] elems, int listItemIndex, LI.Remodel remodel) {
```

Comment:
This comment is generally about going to static. I guess it's totally safe when the methods are private. I have a long standing nervousness about using static in some of these cases. I guess it stems from the inheritance issues. But I like the added info you get from declaring static, it references nothing in the instance. Kind of like how I feel about not doing `int i = someVal` if the assignment is never used.